### ISSUE TITLE
New line

### ISSUE THREAD

**Issue by pA1nD** [webdesignio/inplace#2]
Newlines should be supported.

**Comment by domachine** [comment]
I don't think so 😄 . To be honest, there's no generic way to represent newlines. It's not clear to render a `</p>` or a `</br>`. This really depends on the use-case, I think and should therefore not be part of this component.
